generators/ivtest

```diff
@@ -216,19 +214,16 @@ for l in ivtest_lists:
                 continue
 
             type_ = ''
-            for t in type_should_fail:
+            for t in type_list:
                 if re.match(t, line[1]):
-                    should_fail = 1
                     type_ = ':type: simulation'
```

Comment:
I think those tests should still be marked with `should_fail_because` because `CE` and `RE` are test types that are supposed to fail:
https://github.com/steveicarus/ivtest/blob/1fa31c0e881290cddb2363b6ef52256fffc56337/regress-ivl1.list#L40-L44

Reply:
Ok, so what the "because" for CE and RE tests, respectively?

Reply:
Ok, I get it, compile error and runtime error.
I'll fix this.